docs/userguide.rst

```diff
@@ -303,7 +303,7 @@ environment spawned on the remote containerised platform.
 
 .. code-block:: console
 
-   $ reana-client open -w myanalysis.1 jupyter
+   $ reana-client open -w myanalysis.1
```

Comment:
Hmm, I'm not sure about removing "jupyter", since "open -w myanalysis.42" can give an impression that this will open the web page for the given analysis, which we don't have yet but which will come.  It'll be good to keep the option open to use "open" without qualifier to point to the future `reana.cern.ch/myanalysis/42` page with workflow details.

This is akin to "hub" command-line client where "hub browse" opens the GitHub repository web page; similarly "reana-client open" can in the future open the REANA web page for the given workflow run.)

Perhaps revert back to having "jupyter" mandatory in order not to loose this possibility?